tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -106,6 +106,7 @@ def test_default_cpu_features(test_microvm_with_api, network_config):
     _check_cpu_features_arm(vm)
 
 
+@pytest.mark.kernel_patch
```

Comment:
Maybe mark them as `nonci`, since that option/marker already exists.

Reply:
replaced with selection in cpu_template fixture